vars/promoteArtifacts.groovy

```diff
@@ -80,5 +86,12 @@ void call(Map args = [:]) {
             path: "releases/$bundleFullPath/",
             workingDir: "$WORKSPACE/artifacts/$artifactPath/dist/$filename/",
             includePathPattern: "**/${filename}-${version}*")
+
+        // upload build and dist
+        s3Upload(
```

Comment:
Could we add example url to comments to help people get context easily

Reply:
The `path` parameter already indicates the url structure